pycaret/classification.py

```diff
@@ -187,14 +187,21 @@ def setup(
         If None, no imputation of missing values is performed.
 
 
-    numeric_imputation: str, default = 'mean'
-        Missing values in numeric features are imputed with 'mean' value of the feature
-        in the training dataset. The other available option is 'median' or 'zero'.
+    numeric_imputation: int, float or str, default = 'mean'
```

Comment:
@tvdboom Where does `iterative` imputation come into picture? How does this relate to `imputation_type`?

Reply:
That didn't change wrt v2.x. There is a parameter in setup `imputation_type` that can be simple or iterative. If simple, you use the parameters `numeric_imputation` and `categorical_imputation`. If iterative, you use the parameters `iterative_imputation_iters`, `numeric_iterative_imputer` and `categorical_iterative_imputer`. I'll add to the docstring that they are ignored when imputation_type=iterative.

Reply:
Looks good, but note that the same documentation will have to change in the Object-Oriented Interface setup method as well

Reply:
Could you also look at SOM algorithm?